frontend/src/components/object/ObjectTag.vue

```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+import Button from 'primevue/button';
+import type { ObjectTag } from '@/interfaces';
+
+defineProps<{
+  objectTag: ObjectTag;
+}>();
+</script>
+
+<template>
+  <div class="grid">
```

Comment:
Tyler has decided that if no tags exist to hide this template completely.
i think in this div you can do: `<div v-if="objectTag.Tag.tagset.length">`